java/.idea/jarRepositories.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Typically we just gitignore the entire `.idea` dirs for internal dev, I think.

Reply:
done